CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cmake_minimum_required(VERSION 2.8.12)
+project(absl)
+
+# enable ctest
+include(CTest)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake)
+
+include(GNUInstallDirs)
+include(ReleaseDebugAutoFlags)
+include(AbseilHelpers)
+
+
+# config options
+option(ABSL_ENABLE_PIC "Enable / disable position-independent code generation" OFF)
+option(ABSL_ENABLE_BUNDLE "Enable / disable embbedded dependencies" ON)
+set(ABSL_STD_CXX_FLAG "${CMAKE_CXX_STD_CXX11}" CACHE STRING "c++ std flag (default: c++11)")
+
+
+include(AbseilFindDependencies)
+
+
+# include current path
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+# -std=X
+set(CMAKE_CXX_FLAGS "${ABSL_STD_CXX_FLAG} ${CMAKE_CXX_FLAGS}") 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_WARNING_VLA} ${CMAKE_CXX_FLAGS} ")
+
+# -fPIC
+if(ABSL_ENABLE_PIC)
```

Comment:
This variable shouldn't be needed, projects can use the variable [CMAKE_POSITION_INDEPENDENT_CODE](https://cmake.org/cmake/help/v3.10/variable/CMAKE_POSITION_INDEPENDENT_CODE.html#variable:CMAKE_POSITION_INDEPENDENT_CODE) if they need it.

Reply:
Already complainted about this, here is the answer:
- https://github.com/abseil/abseil-cpp/pull/8#discussion_r142859869

I still think it should not be here but I wanted to mention it.